### PR TITLE
Web - Likes

### PR DESCRIPTION
**Pull request by Rikimbili**
**The PR `Web - Add Posts to Collections` must be reviewed and merged before this is reviewed as this branch extends that one**

### CODE REVIEW

**Review by rob-3 — CHANGES_REQUESTED** [review]
Lgtm, although the approach of just shipping the entire list of likes over the wire is something I'm suspicious of

File: web/src/components/Layout/ImageList.tsx [hunk 1]
```diff
@@ -23,6 +24,7 @@ export default function ImageList({
   areMorePostsLoading,
   className = "",
 }: Props) {
+  console.log(posts);
```
[the remark this answers is not in the document]
Comment (Rikimbili) [reply web/src/components/Layout/ImageList.tsx:27]: nah, that was left during testing lol. Good catch

**Comment by Rikimbili** [comment]
> Lgtm, although the approach of just shipping the entire list of likes over the wire is something I'm suspicious of

We are also sending all posts belonging to a collection when fetching that collection. It might not be the best solution but it is the fastest one to implement and I dont expect us to scale to hundreds or thousands anytime soon

**Comment by rob-3** [comment]
> > Lgtm, although the approach of just shipping the entire list of likes over the wire is something I'm suspicious of
> 
> We are also sending all posts belonging to a collection when fetching that collection. It might not be the best solution but it is the fastest one to implement and I dont expect us to scale to hundreds or thousands anytime soon

Yep, just something to keep in mind.

**Comment by rob-3** [comment]
Remove the log and I'll approve this

**Review by rob-3 — APPROVED** [review]

**Pull request merged**